tests/test_nodes.py

```diff
@@ -353,22 +354,36 @@ def test_legal_assignment_enum(
                 node_type(enum_type)
 
 
-def test_pretty_with_enum() -> None:
+def test_to_yaml_with_enum() -> None:
     cfg = OmegaConf.create()
     assert isinstance(cfg, DictConfig)
     cfg.foo = EnumNode(Enum1)
     cfg.foo = Enum1.FOO
 
     expected = """foo: FOO
 """
-    s = cfg.pretty()
+    s = OmegaConf.to_yaml(cfg)
     assert s == expected
     assert (
         OmegaConf.merge({"foo": EnumNode(Enum1, value="???")}, OmegaConf.create(s))
         == cfg
     )
 
 
+def test_pretty_deprecated() -> None:
+    c = OmegaConf.create({"foo": "bar"})
+    with pytest.warns(
+        expected_warning=UserWarning,
+        match=re.escape(
+            """
+            pretty() is deprecated, use OmegaConf.to_yaml() and resolve
+            now defaults to True (Since 2.0.1)
+            """
+        ),
+    ):
+        assert c.pretty() == "foo: bar\n"
+
+
```

Comment:
not sure this is the vest place for it. why test_nodes?

Reply:
Since the abstract method was in Container I thought it would be better but now looking at it it doesn't make that much sense. I'll move everything to test_omegaconf as you said since it's where to_yaml is.